tock/tock/static/sass/_header.scss

```diff
@@ -0,0 +1,101 @@
+// To mimic behavior of primary nav dropdown
+@include at-media($theme-header-min-width) {
+  .usa-nav__secondary {
+    button,
+    a {
+      padding: 1rem;
+      display: inline-block;
+    }
+    
+    .usa-accordion__button {
+      background-color: transparent;
+      width: auto;
+      font-weight: normal; 
+      color: $theme-color-base;
+      font-size: 0.88rem;
+      background-repeat: no-repeat;
+      background-size: 0.5rem;
+      background-position: right 1rem top 53%;
+      
+      &[aria-expanded="false"] {
+        background-image: url("../../vendor/uswds_v2.3.1/img/angle-arrow-down.svg"), linear-gradient(transparent, transparent);
+      }
+
+      &[aria-expanded="true"] {
+        background-image: url("../../vendor/uswds_v2.3.1/img/angle-arrow-up-white.svg"), linear-gradient(transparent, transparent);
+        -moz-osx-font-smoothing: grayscale;
+        -webkit-font-smoothing: antialiased;
+        background-color: $theme-color-primary-darker;
+        color: white;
+      }
+      
+      span {
+        padding-right: 1rem;
+      }
+    }
+
+    .usa-nav__secondary-links .usa-nav__secondary-item {
+      padding-left: 0; // uswds override
+    }
+    
+    .usa-nav__submenu {
+      margin-top: -3px;
+    }
+  }
+}
+
+@include at-media-max($theme-header-min-width) {
+  .usa-header--extended .usa-nav__secondary-links ul[aria-expanded="true"] {
+    position: relative;
+  }
+  .usa-nav__secondary-item {
+    border-top: 1px solid #dcdee0;
```

Comment:
This is the only place I can find this hex code used anywhere, so I just hardcoded it. I feel dirty.